t/02-charges.t

```diff
@@ -49,6 +49,16 @@ subtest 'create charge' => sub {
         metadata    => { bar => 'baz' },
     }),
         '... Updated charge data';
+
+    my $charges = stripe->get_charges;
```

Comment:
Does this pass? Seems like it would fail on HashRef check... since it would be undef. I think you need Maybe[HashRef] :$query on the calls